care/facility/models/patient.py

```diff
@@ -766,3 +761,130 @@ class PatientNotes(FacilityBaseModel, PatientRelatedPermissionMixin):
         null=True,
     )
     note = models.TextField(default="", blank=True)
+
+
+class PatientHealthDetails(PatientBaseModel, PatientRelatedPermissionMixin):
+    patient = models.ForeignKey(
+        PatientRegistration,
+        on_delete=models.CASCADE,
+        related_name="health_details",
+    )
+    facility = models.ForeignKey(
```

Comment:
consultation is already 1-1 tied to a facility. consultation is also tied 1-1 to a patient